complete application 1-10

rhcaeb opened the pull request:
# Application 1-10

### Notes

pull-request comparing master and app1-10.R branch; 

pending review @belinskyc

rhcaeb committed:
complete application 1-10

belinskyc commented on _scripts/app1-10.R:
```diff
@@ -0,0 +1,38 @@
+  {
+    cat("Category II\n");
+  }
+  else if(fishAge > 5) # age > 5 && port == "north" || "south"
```
If I type in "west" as the location (or "east" or "Timbuktu") and age=7 then this codeblock will execute.

belinskyc commented on _scripts/app1-10.R:
```diff
@@ -0,0 +1,38 @@
+  {
+    cat("Category III\n");
+  }
+  else if(fishAge < 2) # age < 2 && port == "north" || "south"
```
Same comment as line 25 -- except age=1

rhcaeb replied:
@belinskyc

would the \n be required in this case if the statement only outputs one line of code in the console? or is it a situation of good practice

belinskyc replied:
Mostly good practice just in case you add more output to the Console somewhere later in your code.  Then the next output will have a fresh, clean line as opposed to being put right after "Category #"

rhcaeb commented on _scripts/app1-10.R:
```diff
@@ -0,0 +1,38 @@
+  {
+    cat("Invalid port location! Please check spelling.");
+  }
+  
```
@belinskyc 

I think these lines of code (33-36; invalid port location message) need to be moved between category II and category III to fix your comments above.

rhcaeb replied:
@belinskyc

made changes to app1-10.R branch;

reviewed the changes and it solves the > 8 years old, but location != south || !=north (e.g. "west" in your example).

rhcaeb committed:
Update app1-10.R

Move port location error message (cat("")) between category II and category III;

belinskyc commented:
There is still one circumstance in which a valid age and location of a fish is entered but no output is given (i.e., the fish is not categorized).  I'll let you see if you can figure it out.
